configs/injectLocalConfigs.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+const path = require("path");
+
+function injectLocalConfigs(sharedConfigs, pathToWidgetRoot) {
+  const localConfigsFileName = "local.config.json";
+  const pathToLocalConfig = path.join(pathToWidgetRoot, localConfigsFileName);
+  if (fs.existsSync(pathToLocalConfig)) {
+    const localConfigs = require(pathToLocalConfig);
+    return Object.assign({}, sharedConfigs, {config: localConfigs});
+  }
+  return sharedConfigs;
+}
+
+module.exports = injectLocalConfigs;
```

Comment:
Indents should be 4 spaces for tab. Do can we apply the same linting and prettier settings as our project... to keep to somehow similar